src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -383,10 +174,11 @@ const AccountClosure = (() => {
         });
     };
 
-    const showErrorPopUp = async (response, $account_closure_error) => {
+    const showErrorPopUp = async (response) => {
         const mt5_login_list = (await BinarySocket.wait('mt5_login_list')).mt5_login_list;
         // clear all previously added details first
-        $account_closure_error.find('.account-closure-details').remove();
+        const $error_modal = $('#account_closure_error');
+        $error_modal.find('.account-closure-details').remove();
```

Comment:
can we change these to stick to vanilla JS since the rest of the file is using that and this PR is improving the existing code and not just implementing the UI change as per scope of the card?